docs/reading/eth2.md

```diff
@@ -95,7 +95,7 @@ This page serves material necessary to catch up with the current state of ETH2 d
 
 ## Ethereum client implementations
 
-* [Geth](https://github.com/ethereum/go-ethereum) \(known also as go-ethereum\) is the Golang implementation of [ETH1](../glossaries/terminology.md#eth1)
+* [Geth](https://github.com/ethereum/go-ethereum) \(known also as go-ethereum\) is the Golang implementation of [ETH1](/docs/glossaries/terminology#eth1)
```

Comment:
/docs/terminology/#eth1

Reply:
yep. I was just flagging a few issues I came across reading the docs. Didn't intend for a comprehensive update :-)  So.. I'm new at this ... should I close this PR or leave it open for other updates?

Reply:
I didn't mean to double-up on the work. I should have checked for existing PR's before i started.  Your work is much appreciated! I see you caught some bad anchors that I missed, need to make sure those get in.